src/main/java/hu/akarnokd/rxjava/interop/FlowableV2ToObservableV1.java

```diff
@@ -57,7 +57,9 @@ public void call(rx.Subscriber<? super T> t) {
 
         @Override
         public void request(long n) {
-            io.reactivex.internal.subscriptions.SubscriptionHelper.deferredRequest(this, requested, n);
+            if (n > 0) {
+                io.reactivex.internal.subscriptions.SubscriptionHelper.deferredRequest(this, requested, n);
```

Comment:
One way to cause this is by using Rx 1.x filter which requests zero items: https://github.com/ReactiveX/RxJava/blob/010256b0e19bdb18f136f80a4dc48c0a53c3da41/src/main/java/rx/internal/operators/OnSubscribeFilter.java#L59

`SubscriptionHelper` in Rx 2 explicitly does not support this: https://github.com/ReactiveX/RxJava/blob/2.x/src/main/java/io/reactivex/internal/subscriptions/SubscriptionHelper.java#L228